apns_suite_test.go

```diff
@@ -5,8 +5,46 @@ import (
 	. "github.com/onsi/gomega"
 
 	"testing"
+	"time"
 )
 
+type mockConn struct {
```

Comment:
I'd say no with the implementation as it stands – it's not a terribly generic solution. I figured since `Conn` is now an interface, users of the library would write their own